flake8_eradicate.py

```diff
@@ -25,7 +25,7 @@ class Checker(object):
 
     options = None
 
-    def __init__(self, tree, filename: str):
+    def __init__(self, tree, filename: str):    # noqa: D107
```

Comment:
Let's not ignore this one. Let's document what `tree` and `filename` are.
Btw, let's add `ast.AST` type to `tree`